contracts/governance/Staking/modules/StakingWithdrawModule.sol

```diff
@@ -15,6 +15,18 @@ import "./shared/StakingShared.sol";
 contract StakingWithdrawModule is IFunctionsList, StakingShared, CheckpointsShared {
     using SafeMath for uint256;
 
+    event MaxVestingWithdrawIterationsUpdated(uint256 oldMaxIterations, uint256 newMaxIterations);
+
+    /// @dev Struct for direct withdraw function -- to avoid stack too deep issue
```

Comment:
```suggestion
/// @dev Struct for vesting cancelling by governance - to avoid stack too deep issue
```

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/451/commits/fb0984a53a378489f0d8c60b09c7a151a8f7fd4d)